provider/aws/formation/app.json.tmpl

```diff
@@ -137,6 +148,50 @@
   {{ end }}
 {{ end }}
 
+{{ define "caches" }}
+  {{ with .Manifest.Caches }}
+  "CacheSubnetGroup" : {
```

Comment:
I used the `with` in this case to create these settings only if cache is being used. Didn't want to stick them in the range as there is no need to create multiple `ElastiCache::SubnetGroup` for each cache. I'll remove the `with` in Output as I'm drawing a blank there.